tests/Tools/PhpClassComparer.php

```diff
@@ -45,7 +45,7 @@ class PhpClassComparer
      */
     public static function compare(string $phpClassOne, string $phpClassTwo, bool $printDiffs = true): bool
     {
-        $phpParser = (new ParserFactory)->create(ParserFactory::PREFER_PHP7);
+        $phpParser = (new ParserFactory)->createForNewestSupportedVersion();
```

Comment:
Wouldn't we want to use the lowest supported PHP version, e.g. PHP 7.4 in our case?

Reply:
Makes sense.. fixed it.